### PR TITLE
Core 543 mg how works contentful

### PR DESCRIPTION
**Pull request by roger-in-kiva**
how it works section in monthlygood page was updated to be managed in contetful

### CODE REVIEW

File: src/pages/MonthlyGood/MonthlyGoodLandingPage.vue [hunk 1]
```diff
@@ -146,7 +149,7 @@ const pageQuery = gql`
 			}
 		}
 		contentful {
-			entries(contentType: "page", contentKey: "monthlygood")
+			entries(contentType: "page", contentKey: "monthlygood", preview: true)
```
Comment (mcstover) [inline src/pages/MonthlyGood/MonthlyGoodLandingPage.vue:152]: Oops let's get rid of this preview flag please.

File: src/pages/MonthlyGood/HowItWorks.vue [hunk 2]
```diff
@@ -68,16 +44,10 @@ export default {
...
 
-.how-works-section-wrapper {
-	.mg-image {
-		fill: $white;
-		width: 3.125rem;
-		display: block;
-		margin: 0 auto 1rem auto;
-	}
+#rteContent >>> figure{
+	margin-bottom: 0;
```
Comment (mcstover) [inline src/pages/MonthlyGood/HowItWorks.vue:50]: @roger-in-kiva and I have done some diving into Tailwinds and it's not-prose declaration and have some ideas where the extra margin this fixes is coming from. However, the alterations to fix the margins on the bottom of imges embedded in rich text fields will be a much broader change so I've suggested this local hack for now. ref. https://kiva.atlassian.net/browse/VUE-1380 as this is a known issue.

**Review by dyersituations — APPROVED** [review]
Looks good, pending removal of the `preview` flag.

**Comment by dyersituations** [comment]
@roger-in-kiva Check the error coming from the PR UI tests run. Looks like a minor syntax issue.

**Review by dyersituations — CHANGES_REQUESTED** [review]
Looks like a syntax issue holding up the test run, so marking as needs changes.

File: src/pages/MonthlyGood/MonthlyGoodLandingPage.vue (hunk 1 above)
Comment (roger-in-kiva) [reply src/pages/MonthlyGood/MonthlyGoodLandingPage.vue]: Removed

**Comment by roger-in-kiva** [comment]
> @roger-in-kiva Check the error coming from the PR UI tests run. Looks like a minor syntax issue.

Fixed

**Review by dyersituations — APPROVED** [review]

**Review by mcstover — COMMENTED** [review]
👍

**Pull request merged**